spec/requests/set_catkey_spec.rb

```diff
@@ -94,8 +97,8 @@
       context 'with a collection that has no existing catkeys' do
         let(:cocina_model) { build(:collection_with_metadata, id: druid, source_id: 'sul:1234') }
 
-        it 'updates the catkey, trimming whitespace' do
-          patch "/items/#{druid}/catkey", params: { catkey: { catkey: '   12345 ' } }
+        it 'updates the catkey' do
```

Comment:
no trimming of whitespace anymore, it will simply not validate anymore if you do this and refuse to save